File: api-examples/cat/search/python/spatial_search_circle.py
import requests

url = "https://<catalogue-server-ip>/catalogue/v1/search?lat=12.273737&lon=78.37475&radius=200000"

payload = {}
headers= {}

response = requests.request("GET", url, headers=headers, data = payload)

print(response.text.encode('utf8'))

File: api-examples/cat/count/java/spatial_search_circle.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://<catalogue-server-ip>/catalogue/v1/count?lat=18.4&lon=73.9&radius=200000")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();
